monolithic/monolithic-layered/src/test/java/com/epam/training/microservices/monolithic/web/MonolithicErrorControllerTest.java

```diff
@@ -0,0 +1,67 @@
+package com.epam.training.microservices.monolithic.web;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.*;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class MonolithicErrorControllerTest {
+
+    @Autowired
+    private TestRestTemplate testRestTemplate;
+
+    @Test
+    @DisplayName("Test for 404 not found page")
+    public void shouldReturnNotFoundPage() {
+        ResponseEntity<String> response = testRestTemplate.getForEntity("/symptom/404", String.class);
+
+        assertThat(response.getStatusCode()).isEqualTo(HttpStatus.NOT_FOUND);
+        assertThat(response.getBody()).contains("<h4 class=\"alert-heading\"><i class=\"bi-exclamation-octagon-fill\"></i>Oops! Page not found</h4>");
+        assertThat(response.getBody()).contains("<p>Sorry, we couldn't find the page you were looking for.</p>");
+    }
+
+    @Test
+    @DisplayName("Test for internal server error page")
+    public void shouldReturnInternalServerErrorPage() {
+        String name = "Value longer than 255 characters cannot handled by database." +
```

Comment:
Oh gosh! It seems like you discovered a new bug. Please create a ticket for it. Or I can do it myself, but later. Thank you!

Reply:
Bug created #52. Test also updated so it don't need to be fixed after bug fixing.